bin/upscaler.py

```diff
@@ -147,7 +150,7 @@ def _upscale_frames(self):
         self.upscaler_exceptions = []
 
         # initialize waifu2x driver
-        if self.waifu2x_driver != 'waifu2x_caffe' and self.waifu2x_driver != 'waifu2x_converter':
+        if self.waifu2x_driver != 'waifu2x_caffe' and self.waifu2x_driver != 'waifu2x_converter' and self.waifu2x_driver != 'waifu2x-ncnn-vulkan':
```

Comment:
This can be changed to

```python
drivers = ['waifu2x_caffe', 'waifu2x_converter', 'waifu2x_ncnn_vulkan']
if self.waifu2x_driver not in drivers:
    raise Exception(f'Unrecognized waifu2x driver: {self.waifu2x_driver}')
```

Also, driver's name should use underscores instead of dashes to be consistent.